react/features/welcome/components/WelcomePageSideBar.native.js

```diff
@@ -0,0 +1,167 @@
+// @flow
+
+import React, { Component } from 'react';
+import { SafeAreaView, ScrollView, Text } from 'react-native';
+import { connect } from 'react-redux';
+
+import SideBarItem from './SideBarItem';
```

Comment:
Why is this component not together with the SideBar in base/react ?

Reply:
I don't feel it too generic. This is an item component that renders welcome page sidebar button-like components, while the SideBar component in base/react is so generic, that it's basically totally empty, and can even contain video, or anything.